sdv/single_table/base.py

```diff
@@ -374,6 +366,32 @@ def _set_random_state(self, random_state):
         """
         self._model.set_random_state(random_state)
 
+    def _randomize_samples(self, randomize_samples):
```

Comment:
I moved this originally so that PAR could use it, but the code seems to call `set_random_state` on the underlying model. `PAR` doesn't have this ability right now. If it is hard to add I might just remove all randomization related stuff from PAR until we can support it